src/data/codingcontracttypes.ts

```diff
@@ -270,14 +269,16 @@ export const codingContractTypesMetadata: ICodingContractTypeMetadata[] = [
       let l = 0;
       let r: number = n - 1;
       let k = 0;
-      while (true) {
+      let done = false;
+      while (!done) {
         // Up
         for (let col: number = l; col <= r; col++) {
           spiral[k] = data[u][col];
           ++k;
         }
         if (++u > d) {
-          break;
```

Comment:
What was wrong with the breaks here? I liked the old way better.

Reply:
Linter didn't like us using a constant as the condition for a loop, this was my attempt at satisfying that.